src/Home/index.tsx

```diff
@@ -264,7 +264,7 @@ const Home: FunctionComponent = () => {
         <SettingsButton />
         <ExposureDetectionStatus />
         <ShareLink />
-        {displayCovidData && <COVIDDataDashboard />}
+        {displayCovidData && <CovidDataClip />}
```

Comment:
```suggestion
        {displayCovidData && <CovidData />}
```

What are your thoughts on just using `CovidData` vs. `CovidDataClip`?

Reply:
`CovidData` is the name of the type we picked. This component is not displaying that per-se is just showing a 'clip' of that data. Does that make sense?

Reply:
Ah I see, that works!